frontend/src/components/shared/card.js

```diff
@@ -1,14 +1,43 @@
 import * as React from "react";
 
 const Card = ({ img, date, title, children, containerCss }) => {
+  function formatDateAndTime(isoDateString) {
```

Comment:
Here is an easy to format graphql date data: https://www.gatsbyjs.com/docs/graphql-reference/#dates

Reply:
I just tried to implement that but it seems to mess with a lot of the event-processing code in event-processing-util.js with the way that the date data is formatted

Reply:
okay.